x-pack/plugins/ml/public/application/components/scatterplot_matrix/scatterplot_matrix.scss

```diff
@@ -0,0 +1,6 @@
+.mlScatterplotMatrix {
+  .vega-bind span {
+    font-size: $euiSizeM;
```

Comment:
To better future proof this, you can use the `$euiFontSizeXS` variable here instead (also 12px). This way, the text here will adjust in line with underlying changes to EUI font styles as opposed to size/layout styles.

Reply:
Thanks for the hint, fixed in 2cd54b25a5e.